internal/integration_test/vs/wasmedge/wasmedge_test.go

```diff
@@ -24,6 +24,30 @@ func TestBenchmarkAllocation_Call_CompilerFastest(t *testing.T) {
 	vs.RunTestBenchmarkAllocation_Call_CompilerFastest(t, runtime())
 }
 
+func TestAllocationCGO(t *testing.T) {
```

Comment:
please keep only the existing Test* Benchmark* functions in the vs package. drop the ones used for the investigation

Reply:
Ah ok, so you don't want any of the cgo things to be benchamrked, correct ?

Reply:
we don't need to add new benchmarks (in vs). we did ones in the interim to make sure nothing regressed with the new advice, that's all.

Reply:
done